support-frontend/assets/pages/[countryGroupId]/thankYou.tsx

```diff
@@ -157,7 +158,7 @@ export function ThankYou({
 	return (
 		<ThankYouComponent
 			geoId={geoId}
-			appConfig={appConfig}
+			csrf={csrf}
```

Comment:
This was unavailable in the thankyou component tests so has been drilled down from here instead